jquery.flot.touchNavigate.js

```diff
@@ -60,43 +13,93 @@ can set the default in the options. */
     function initTouchNavigation(plot, options) {
         var startPageX = 0,
             startPageY = 0,
-            plotState = false;
-
-        function saveNavigationData(plot, e) {
-            if (e.touches && e.touches[0]) {
-                var opts = plot.getOptions();
-                opts.navigationData = {
-                    lastClientX: e.touches[0].clientX,
-                    lastClientY: e.touches[0].clientY
-                }
-            }
+            plotState = false,
```

Comment:
plotState is not a boolean. Don't initialize it with false. Don't initialize it at all.